Libraries/CameraRoll/RCTImagePickerManager.m

```diff
@@ -65,7 +65,7 @@ - (dispatch_queue_t)methodQueue
   imagePicker.sourceType = UIImagePickerControllerSourceTypeCamera;
 
   if ([RCTConvert BOOL:config[@"videoMode"]]) {
-    imagePicker.cameraCaptureMode = UIImagePickerControllerCameraCaptureModeVideo;
+    imagePicker.mediaTypes = @[(NSString *)kUTTypeImage, (NSString *)kUTTypeMovie];
```

Comment:
From the linked article, it seems you can use:

```
[UIImagePickerController availableMediaTypesForSourceType:
        UIImagePickerControllerSourceTypeCamera]
```

This will choose the right media types for the video mode. Would that work here?

Also - what do you mean by photo mode plus video mode? Is that the camera interface where you can both record a video as well as snap photos during recording? If so, is that actually in the image picker as well?